frontend/src/metabase-types/api/mocks/settings.ts

```diff
@@ -130,9 +131,12 @@ export const createMockSettingDefinition = (
   ...opts,
 });
 
-export const createMockSettings = (opts?: Partial<Settings>): Settings => ({
+export const createMockSettings = (
+  opts?: Partial<Settings | EnterpriseSettings>,
```

Comment:
I copied this from https://github.com/metabase/metabase/blob/085c37800c59d93d9a92abfe334f8f5b212c5022/frontend/test/__support__/settings.ts#L8-L9
But considering the other comment, maybe we can just do 
```suggestion
  opts?: Partial<EnterpriseSettings>,
```